pandas/tests/io/formats/style/test_to_latex.py

```diff
@@ -1032,3 +1032,15 @@ def test_concat_recursion():
     """
     )
     assert result == expected
+
+
+@pytest.mark.parametrize(
+    "df",
+    [
+        DataFrame(),
+        DataFrame(columns=["a", "b", "c"]),
+    ],
+)
+def test_empty_clines(df: DataFrame):
+    # GH 47203
+    df.style.to_latex(clines="all;data")
```

Comment:
Could you assert the latex result of this call?

Reply:
Done and dusted! Good call. I've also added a simple parametrize over all the legal `clines` arguments. Should be all good to go.